library/core/class.request.php

```diff
@@ -1023,7 +1027,13 @@ public function url($path = '', $withDomain = false, $ssl = null) {
                 }
             }
         }
-        $parts[] = ltrim($path, '/');
+
+        // Prevent the addition of webRoot a second time if the path was already encoded.
+        $path = ltrim($path, '/');
```

Comment:
This is going to be a bug. What if I install Vanilla into a directory called "/discussions"?